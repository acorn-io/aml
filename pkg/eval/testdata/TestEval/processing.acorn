a: {
    x: a.b
    b: 1
	c: a.x
}
