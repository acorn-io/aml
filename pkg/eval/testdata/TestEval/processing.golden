{
  "a": {
    "b": 1,
    "c": 1,
    "x": 1
  }
}